PyEMD/visualisation.py

```diff
@@ -74,7 +75,7 @@ def plot_imfs(self, imfs=None, residue=None, t=None, include_residue=True):
         # Making the layout a bit more pleasant to the eye
         plt.tight_layout()
 
-    def plot_instant_freq(self, t, imfs=None):
+    def plot_instant_freq(self, t, imfs=None, order=1, alpha=0.3):
```

Comment:
I'd much rather that the default plotting is done without the filter. For one, this is going to break plots that people already have so we can't do it quietly. Such change should be explicitly added in the README file and in documentation.

The artefacts you see are due to the compact support of discrete time series and not the method itself. The change is helping mitigate the visual pleasantness but it's modifying the results. I honestly think that the user should have an option whether to display results filtered or not, and for now it should be default to not filtering. 

How about leaving all as is but changing defaults to `plot_instant_freq(self, t, imfs=None, order=None, alpha=None)` and revert options when respective values are `None`?

Reply:
Dear @laszukdawid, 

I am very happy to be able to contribute (even if only a little) to the work you have been doing.

Regarding the default values, I totally agree! You made a good point. So I will set the non-filtered plot as default and use `None` values, as suggested.
Also, I will include the test case for the ValueError when you don't run EMD, EEMD, or CEEMDAN.

At this moment I am a little busy, but I will try to do it ASAP. Then, I let you know when I finish making these corrections so you can take a look. It should be done by the end of the week for sure though.

Thank you,
Best